app/models/manageiq/providers/ansible_tower/inventory/parsers/automation_manager.rb

```diff
@@ -0,0 +1,41 @@
+class ManageIQ::Providers::AnsibleTower::Inventory::Parsers::AutomationManager
+  attr_accessor :collector
+
+  def initialize(collector)
+    @collector = collector
```

Comment:
we could pass Inventory object here, or we need to pass more args, collector, target, options, maybe ems.